substratevm/src/com.oracle.svm.reflect/src/com/oracle/svm/reflect/target/Target_jdk_internal_misc_Unsafe_Reflection.java

```diff
@@ -67,4 +68,9 @@ public long objectFieldOffset(Class<?> c, String name) {
             throw new InternalError();
         }
     }
+
+    @Substitute
+    private long staticFieldOffset(Target_java_lang_reflect_Field f) {
+        return f.offset;
```

Comment:
What happens if we don't have the field in the configuration? Don't we need a similar check as in the object version of the method?

Reply:
Fixed.

Reply:
Also added a comment as clarification as to why the implementation is the same.